aiida/cmdline/commands/cmd_computer.py

```diff
@@ -501,43 +491,70 @@ def computer_test(user, print_traceback, computer):
     num_failures = 0
     num_tests = 0
 
+    tests = {
+        _computer_test_no_unexpected_output: 'Checking for spurious output',
+        _computer_test_get_jobs: 'Getting number of jobs from scheduler',
+        _computer_get_remote_username: 'Determining remote user name',
+        _computer_create_temp_file: 'Creating and deleting temporary file'
+    }
+
     try:
-        echo.echo('> Testing connection...')
+        echo.echo('* Opening connection... ', nl=False)
+
         with transport:
```

Comment:
What if an error happens here (e.g. configure a wrong username).
It will start printing stuff...

Reply:
I don't think so, it will be caught in the outer try-except and that also just prints the error or including the traceback if the option is specified